### PR TITLE
Classifier itwinviewer

### PR DESCRIPTION
**Pull request by dustinlebsock**
Updates the classifier sample to use the itwinviewer react component.

### CODE REVIEW

File: src/frontend-samples/classifier-sample/ClassifierWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+  const _onOutsideDisplayChange = (event: React.ChangeEvent<HTMLSelectElement>): void => {
+    setOutsideDisplayKeyState(event.target.value);
+  };
+
+  const _onInsideDisplayChange = (event: React.ChangeEvent<HTMLSelectElement>): void => {
+    setInsideDisplayKeyState(event.target.value);
+  };
+
+  // Display drawing and sheet options in separate sections.
```
Comment (williamkbentley) [inline src/frontend-samples/classifier-sample/ClassifierWidget.tsx:141]: Stray comment

File: src/frontend-samples/classifier-sample/ClassifierApi.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (williamkbentley) [inline src/frontend-samples/classifier-sample/ClassifierApi.ts:1]: Update README to say that this file has the necessary static methods for interacting with iModel.js

File: src/frontend-samples/classifier-sample/ClassifierWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  const _outsideDisplayEntries: { [key: string]: string } = {};
+  _outsideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Off]] = "Off";
+  _outsideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.On]] = "On";
+  _outsideDisplayEntries[SpatialClassificationProps.Display[SpatialClassificationProps.Display.Dimmed]] = "Dimmed";
+
+  const iModelConnection = useActiveIModelConnection();
+  const viewport = useActiveViewport();
+  const [initalized, setInitalized] = React.useState<boolean>(false);
+  const [classifiersState, setClassifiersState] = React.useState<{ [key: string]: string }>({});
+  const [classifierState, setClassifierState] = React.useState<string>();
```
Comment (williamkbentley) [inline src/frontend-samples/classifier-sample/ClassifierWidget.tsx:34]: Maybe you can find better names. `classifiersState` and `classifierState` are a little too similar

**Review by williamkbentley — CHANGES_REQUESTED** [review]

File: src/frontend-samples/classifier-sample/ClassifierWidget.tsx (hunk 3 above)
Comment (dustinlebsock) [reply src/frontend-samples/classifier-sample/ClassifierWidget.tsx]: Renamed to 'classifiers' and 'currentClassifier' respectively.

File: src/frontend-samples/classifier-sample/ClassifierApi.ts (hunk 2 above)
Comment (dustinlebsock) [reply src/frontend-samples/classifier-sample/ClassifierApi.ts:1]: Readme update to point to proper file.

File: src/frontend-samples/classifier-sample/ClassifierWidget.tsx (hunk 1 above)
Comment (dustinlebsock) [reply src/frontend-samples/classifier-sample/ClassifierWidget.tsx]: removed.

**Review by williamkbentley — APPROVED** [review]

**Pull request merged**